src/experimentEnrollment.js

```diff
@@ -4,20 +4,21 @@ export default {
 
   enrollInExperiment(component, child, variationName) {
     if (variationName && child.props.name === variationName) {
-      component.exposedVariation = child;
+      component.selectedVariation = child;
     } else if (child.props.displayName === DEFAULT_EXPERIMENT_COMPONENT) {
       component.defaultComponent = child;
     }
     return component;
   },
-  
-  getExposedExperimentVariation(childrenComponents, variationName) {
+
+  getVariation(childrenComponents, variationName) {
+    // I don't understand what this line does.
     if (!childrenComponents.reduce) {
```

Comment:
perhaps this line is worth a comment...it is incredibly strange behavior that unfortunately seems to come from React itself. If the number of children components is 1, then React does not treat ```this.props.children``` as an iterable object and therefore reduce is not defined on the object making this kind of weird coding necessary.